### ISSUE TITLE
Export landscape layout

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu/yaru_widgets.dart#699]
@Jupi007 @jpnurmi do you have anything against exporting the landscape layout? I could use it without the portrait layout ever showing up

**Comment by Jupi007** [comment]
Why not 🙂
Sidebar items can be adapted to small screens by apps.

**Comment by jpnurmi** [comment]
I recall the fix for #512 (still waiting for the next Flutter 3.x) would change the structure so that there wouldn't be separate portrait and landscape layouts with their own navigators anymore but it would be the one and same navigator in both modes with an optional listview decoration on the side. In that sense, it would be safer to set the breakpoint to 0 or infinity to force either layout to avoid exposing the internal building blocks until we know what the final structure looks like.

**Comment by Feichtmeier** [comment]
> I recall the fix for #512 (still waiting for the next Flutter 3.x) would change the structure so that there wouldn't be separate portrait and landscape layouts with their own navigators anymore but it would be the one and same navigator in both modes with an optional listview decoration on the side. In that sense, it would be safer to set the breakpoint to 0 or infinity to force either layout to avoid exposing the internal building blocks until we know what the final structure looks like.

😵 Ok I needed to re-read this 2 times but I think I got it. Ok then I think we can wait for this. Meanwhile I can experiment with just a navigationrail I think 👍 

I thought about one more layout where you switch between landscape and a portrait layout but without a master navigation point but instead a drawer/hamburger menu to select the details

**Comment by jpnurmi** [comment]
I meant something like this:
```dart
YaruMasterDetailTheme(
  data: const YaruMasterDetailThemeData(breakpoint: 0),
  child: YaruMasterDetailPage(
    // never below the breakpoint so always landscape
  ),
)
```
